### PR TITLE
Upgrade to v1.0.0

### PR DESCRIPTION
**Pull request by hovaesco**
Upgrade dbt-trino to dbt-core v1.0.0.

Note: dbt-labs has not released official docker image yet.

### CODE REVIEW

File: docker/dbt/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM fishtownanalytics/dbt:0.21.0
+FROM xemuliam/dbt:1.0.0
```
Comment (findinpath) [inline docker/dbt/Dockerfile:1]: I would advise against doing this.
Let's wait until the official docker image for `dbt` lands.

File: setup.py [hunk 2]
```diff
@@ -14,55 +14,51 @@
...
-    author='findinpath',
-    author_email='findinpath@gmail.com',
-    url='https://github.com/starburstdata/dbt-trino',
-
-    packages=find_namespace_packages(include=['dbt', 'dbt.*']),
+    long_description_content_type="text/markdown",
+    platforms="any",
+    license="Apache License 2.0",
+    license_files=("LICENSE.txt",),
+    author="findinpath",
```
Comment (findinpath) [inline setup.py:48]: Please replace the `author` with `Starburst Data` and also use a _contact_  Starburst email.

**Comment by findinpath** [comment]
Are there any breaking changes coming with the migration to `1.0.0` for this adapter?
If yes, could you please document them in the `README.md` of the project?

**Review by findinpath — CHANGES_REQUESTED** [review]

File: docker/dbt/Dockerfile (hunk 1 above)
Comment (hovaesco) [reply docker/dbt/Dockerfile:1]: yes I agree, it's used just to not stop the work on the upgrading to v1.0.0 

as an alternative, we can also build our own image

**Comment by hovaesco** [comment]
> Are there any breaking changes coming with the migration to 1.0.0 for this adapter?
If yes, could you please document them in the README.md of the project?

Here is an official migration guide: https://docs.getdbt.com/docs/guides/migration-guide/upgrading-to-1-0-0
I will add any necessary migration tips into `README.md`.

File: docker/dbt/Dockerfile (hunk 1 above)
Comment (hovaesco) [reply docker/dbt/Dockerfile]: Official dbt-core docker image is already avaliable. 
I reverted a previous commit.

File: docker-compose-trino.yml [hunk 3]
```diff
@@ -3,7 +3,7 @@ services:
   trino:
     ports:
       - "8080:8080"
-    image: "trinodb/trino:364"
+    image: "trinodb/trino:365"
```
Comment (findinpath) [inline docker-compose-trino.yml:6]: could you please update to `366` ?

**Review by findinpath — APPROVED** [review]

**Review by findinpath — APPROVED** [review]

**Pull request merged**